### ISSUE TITLE
'unexpected error' while slicing

### ISSUE THREAD

**Issue by esmerel** [Ultimaker/Cura#15245]
### Application Version

5.3

### Platform

MacOS 12.6.5

### Printer

Prusa MK3/MK3s

### Reproduction steps

I literally just upgraded to 5.3. 
I started the slicer.
I loaded an STL, and flipped its orientation.
I changed to the Prusa profile.
Changed the support settings to tree and toggled the adhesion switch to 'on'.
Clicked the slice button.
The "something unexpected" or 'went wrong' dialog popped up and said the slice failed or encountered an error and "You might want to file a bug report!" I *think* it had gotten partway through slicing.

At any rate, clicked the but report button, ended up here.

I tried slicing again and it completed normally and the file is printing.  The app didn't even crash, or require a reload.
[cura.log](https://github.com/Ultimaker/Cura/files/11238148/cura.log)

### Actual results

The "something unexpected" or 'went wrong' dialog popped up and said the slice failed or encountered an error and "You might want to file a bug report!" I *think* it had gotten partway through slicing.

### Expected results

I expected the file to complete slicing :D

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

I honestly don't expect anything to come from this, I'm sending it in literally because it was a brand new install of 5.3 and the log might have something? And not be completely awful to dig through.

**Comment by GregValiant** [comment]
Thanks for the report.
I'm not real good at reading the log files but it appears you had the Octoprint Plugin loaded?  Maybe @fieldOfView has a minute to review this.  He may want/need a Project File ("File | Save Project") as well.  It must be zipped and the zip folder posted.

**Comment by esmerel** [comment]
[nalgene bottle.3mf.zip](https://github.com/Ultimaker/Cura/files/11240245/nalgene.bottle.3mf.zip)
one zipped project file!

**Comment by esmerel** [comment]
and yes, I do have the octoprint plugin loaded, that is correct.

**Comment by nallath** [comment]
Thanks for reporting this! We're aware of an issue that causes some models to not slice. We don't have a fix yet, but we are investigating it.

In some cases it can help to move / rotate the model a bit. Although it's not a real fix, you might try this workaround for the moment

**Comment by esmerel** [comment]
Yep, I literally just tried it a second time and it sliced and printed fine. :D
